### ISSUE TITLE
Does this library still work with Hootsuite's OAuth2 authentication? 

### ISSUE THREAD

**Issue by akoithra** [BclEx/hootsuite-rest#4]
I keep getting rejected Promise errors. Entirely possible I'm doing something dumb - thanks!

**Comment by jnagorka** [comment]
No, it doesn't look like the Node client has been updated with the new auth changes yet. However the .NET client does support it.

**Comment by kennetherland** [comment]
I got it to work via this article:
https://github.com/googlesamples/oauth-apps-for-windows/blob/master/OAuthConsoleApp/OAuthConsoleApp/Program.cs#L80
